server/parsec/cli/testbed.py

```diff
@@ -40,24 +40,24 @@ class UnknownTemplateError(Exception):
     pass
 
 
+type Template = tuple[OrganizationID, int, testbed.TestbedTemplateContent]
```

Comment:
Use `TypeAlias` for the moment, `type` is still a bit too fresh for our IDEs

Reply:
done